### DISCUSSION TITLE
Is there any way to push attachments to Vendors?

### DISCUSSION THREAD

**Discussion by abeerag** [codatio/codat-docs#668] [Q&A]
Hi, We'd like to push attachments to vendors in QBO, Xero and Netsuite - is this functionality supported?

**Comment by mcclowes** [comment]
We call these (Vendors in QBO, Contacts in Xero) [suppliers](https://docs.codat.io/accounting-api#/schemas/Supplier).

Although we don't support attachments against suppliers in our standardized data model, we do have a [supplemental data feature](https://docs.codat.io/using-the-api/supplemental-data/overview) which can be used to access QBO/Xero-specific fields and should achieve what you need here.

Netsuite isn't currently supported by our Supplemental Data feature.
